homeassistant/components/denonavr/media_player.py

```diff
@@ -247,12 +247,47 @@ def __init__(
             and MediaPlayerEntityFeature.SELECT_SOUND_MODE
         )
 
+        self._receiver.register_callback("ALL", self._telnet_callback)
```

Comment:
Entity state update callbacks should be registered in the entity coroutine method `async_added_to_hass`.